### ISSUE TITLE
Serial port read failure

### ISSUE THREAD

**Issue by mehtajaghvi** [ros-drivers/rosserial#186]
I am using ultrasound example provided in ros_lib.
I am running that using rosrun rosserial_python serial_node.py at 57600 baud rate. This results in serial port read failure. 

[WARN] [WallTime: 1402665687.484972] Serial Port read returned short (expected 1 bytes, received 0 instead).
[WARN] [WallTime: 1402665687.485703] Serial Port read failure

I have tried using rosserial from git, from binary, changed baud rates, changed ports, re-installed ros_lib.
The last edit I did was change the timeout from 5 to 10 in serilclient.py, this does reduce the frequency of error.

**Comment by miguel7676** [comment]
Hi there, have you had any luck sorting this issue out?

**Comment by mehtajaghvi** [comment]
No, I could not find any answers. It does not affect the system as it is more of a warning but this happens with not just one sensor but multiple different kinds. I tried everything possible over internet but to no avail

**Comment by miguel7676** [comment]
Oh ok thank you for the reply. Ive also searched the Internet and found nothing. Just some hints that it is the arduino buffers that are too full.
